org.eclipse.jdt.ls.tests/projects/maven/salut/src/main/java/java/Foo4.java

```diff
@@ -0,0 +1,32 @@
+package java;
```

Comment:
please avoid using the `java` package for new test code. It was used in the initial tests but turned out to be a mistake, since it's not an authorized package name

Reply:
Sure.